rest-api-spec/src/main/resources/rest-api-spec/api/reindex.rethrottle.json

```diff
@@ -0,0 +1,24 @@
+{
+  "reindex.rethrottle": {
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/plugins/master/plugins-reindex.html",
+    "methods": ["POST"],
+    "url": {
+      "path": "/_reindex/{task_id}/_rethrottle",
+      "paths": ["/_reindex/{task_id}/_rethrottle", "/_update_by_query/{task_id}/_rethrottle"],
+      "parts": {
+        "task_id": {
+          "type": "string",
+          "description": "The task id to rethrottle"
+        }
+      },
+      "params": {
+        "requests_per_second": {
+          "type": "float",
+          "default": 0,
```

Comment:
I can make whatever you think is right work. I don't really have an opinion. I can also tell the user it is an error to set the parameter to empty string.